File: reto14/reto14.js
/**
 * @param {number[]} reindeer
 * @param {number[]} stables
 * @returns {number}
 */
function minMovesToStables(reindeer, stables) {
  reindeer.sort();
  stables.sort();
  let moves = 0;
  for (let i = 0; i < reindeer.length; i++) {
    moves +=
      Math.max(reindeer[i], stables[i]) - Math.min(reindeer[i], stables[i]);
  }
  return moves;
}

File: reto14/reto14.py
def min_moves_to_stables(reindeer, stables):
    reindeer.sort()
    stables.sort()
    moves = 0
    for i in range(len(reindeer)):
        moves += max(reindeer[i], stables[i]) - min(reindeer[i], stables[i])
    return moves
